tools/iree-e2e-matmul-test.c

```diff
@@ -1056,17 +1056,18 @@ static iree_status_t replay_event_call_matmul(iree_trace_replay_t* replay,
  *****************************************************************************/
 
 // Helper for |replay_event_requirements|.
-static iree_status_t iree_cpu_features_have_all_target_features(
-    iree_cpu_features_t* cpu_features, yaml_document_t* document,
-    yaml_node_t* target_features_node) {
+static iree_status_t iree_cpu_has_required_target_features(
+    yaml_document_t* document, yaml_node_t* target_features_node) {
   for (yaml_node_item_t* item = target_features_node->data.sequence.items.start;
        item != target_features_node->data.sequence.items.top; ++item) {
     yaml_node_t* item_node = yaml_document_get_node(document, *item);
     iree_string_view_t required_feature = iree_yaml_node_as_string(item_node);
     if (iree_string_view_is_empty(required_feature)) continue;
-    bool feature_is_supported = false;
-    IREE_RETURN_IF_ERROR(iree_cpu_features_query(cpu_features, required_feature,
-                                                 &feature_is_supported));
+    int64_t feature_is_supported = 0;
+    fprintf(stderr, "Looking up feature by key: %.*s\n",
+            (int)required_feature.size, required_feature.data);
```

Comment:
Probably want to remove this `fprintf`, so it doesn't appear in test / CI output for every run?

Reply:
woops, thanks for the catch!

Reply:
done